copter/source/docs/traditional-helicopter-connecting-apm.rst

```diff
@@ -78,22 +69,17 @@ RC Calibration
    gearbox. If you have a piston engine helicopter, also disconnect the throttle
    servo linkage. 
 
-The RC MUST be calibrated before proceeding once the autopilot is powered up.
-RC calibration is identical to all other vehicles. With helicopters using the
-ArduPilot system there can be no mixes in the RC radio. All the outputs must be
-"pure", i.e. use either airplane mode in your radio, or helicopter mode with H1
-or "straight" swash.
+The RC MUST be calibrated before proceeding once the autopilot is powered up. RC calibration is identical to all other vehicles. With helicopters using the ArduPilot system there can be no mixes in the RC radio. All the outputs must be
+"pure", i.e. use either airplane mode in your radio, or helicopter mode with H1 or "straight" swash.
 :ref:`See this topic <common-radio-control-calibration>`.
 
 Compass Calibration
 -------------------
 
-It is recommended to calibrate the compasses at this time as well. This is the
-same as all other vehicles.
+It is recommended to calibrate the compasses at this time as well. This is the same as all other vehicles.
 :ref:`See this topic <common-compass-calibration-in-mission-planner>`.
 
 Accelerometer Calibration
 -------------------------
-If the accelerometers were not calibrated on the bench prior to installation it
-must be calibrated before proceeding.
+If the accelerometers were not calibrated on the bench prior to installation it must be calibrated before proceeding.
```

Comment:
It might be worth a note here that accelerometer calibration in helicopters is very difficult with the controller mounted in the frame. With large 800 class machines that weigh 30 lbs or more, it almost impossible. Highly recommended to perform accel calibration before installing the flight control. It can be done with the control connected to the GCS with USB cable before installation.

Reply:
I even do that on my small planes and multicopters....its always easier to cal first time on the bench....with Tridge's new align on a landmark and enter the heading via gcs one button cal for compass, doing the compass cal with a 30lb heli will get a lot easier also...